nav2_navfn_planner/include/nav2_navfn_planner/navfn_planner.hpp

```diff
@@ -43,6 +43,8 @@ class NavfnPlanner : public nav2_util::LifecycleNode
   NavfnPlanner();
   ~NavfnPlanner();
 
+  bool has_nested_service_requests() {return has_nested_client_;}
```

Comment:
It'd be nice if there was a function you could call on the node to return a count of service clients it has, so that we don't have to define that here.